cicd/pipelines/configuring-security-context-for-pods.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="configuring-security-context-for-pods"]
+= Configuring the security context for pods
+include::_attributes/common-attributes.adoc[]
+:context: configuring-security-context-for-pods
+
+toc::[]
+
+The default service account for pods that {pipeline-shortnames} starts is `pipeline`. The security context constraint (SCC) associated with the `pipeline` service account is `pipelines-scc`. The `pipelines-scc` SCC is based the `anyuid` SCC, with minor differences as defined in the following YAML specification:
+
+.Example `pipelines-scc.yaml` snippet
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+...
+allowedCapabilities:
+  - SETFCAP
+...
+fsGroup:
+  type: MustRunAs
+...
+----
+
+In addition, the `Buildah` cluster task, shipped as part of {pipelines-shortname}, uses `vfs` as the default storage driver.
+
+You can configure the security context for pods that {pipelines-shortname} creates for pipeline runs and task runs. You can make the following changes:
+
+* Change the default and maximum SCC for all pods
+* Change the default SCC for pods created for pipeline runs and task runs in a particular namespace
+* Configure a particular pipeline run or task run to use a custom SCC and service account
+
+[NOTE]
+====
+The simplest way to run `buildah` that ensures all images can build is to run it as root in a pod with the `privileged` SCC. For instructions about running `buildah` with more restrictive security settings, see xref:../../cicd/pipelines/unprivileged-building-of-container-images-using-buildah.adoc#unprivileged-building-of-container-images-using-buildah[Building of container images using Buildah as a non-root user].
+====
+
+include::modules/op-configuring-default-maximum-scc.adoc[leveloffset=+1]
```

Comment:
Please add a line break after this include statement.

`// Add a blank line after each 'include::' statement.` Source: the [modular docs assembly template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc)